api/devpost_scraper.py

```diff
@@ -0,0 +1,50 @@
+from bs4 import BeautifulSoup
+import urllib.request as ur
+from functools import reduce 
+
+def get_challenges(link):
+    sock = ur.urlopen(link)
+    soup = BeautifulSoup(sock,"html.parser")
+
+    #Find the prizes
+    filters = soup.find("form",{"class":"filter-submissions"}).findAll("ul")
+    prize_list = filters[0].findAll("label")
+    prize_ids = [prize.find("input")["value"] for prize in prize_list]
+    prize_list = [row.contents[-1] for row in prize_list]
+
+    if(link[-1]!="/"):
+        link+="/"
+
+    num_winners = []
+
+    #Find the number of winners for each prize
```

Comment:
Ah shoot, I should've been more clear with this part -- so the # of winners for a project isn't actually what is on Devpost. This number is how many prizes the company intends to give out. So the timeline goes like this: company gives us info (including the number of prizes they want to give out) -> we create the Devpost stuff -> _then_ we import into hackathon-expo-app using your code. Default number of prizes to award is 1, but and otherwise Devpost includes it in the challenge name.

For example, in this image, Ark is giving out 5, 1517 Fund is giving out 2, and Booz Allen is giving out 1.
![image](https://user-images.githubusercontent.com/14797288/52237375-59023380-2897-11e9-9a9e-54e7679a51e4.png)

Reply:
So I guess a better variable name instead of `num_winners` would be something like `number_potential_winners` but let's not worry about renaming - they would end up being the same number anyways :)